components/expenses/ExpenseForm.js

```diff
@@ -194,6 +195,7 @@ export const prepareExpenseForSubmit = expenseData => {
     privateMessage: expenseData.privateMessage?.trim(),
     invoiceInfo: expenseData.invoiceInfo?.trim(),
     reference: expenseData.reference?.trim(),
+    recipientNote: expenseData.recipientNote?.trim(),
```

Comment:
After this addition, I get an error if I try to update a draft invite: 
![image](https://github.com/user-attachments/assets/db32d344-e0bc-492c-9570-32aa18001b29)

Related: https://github.com/opencollective/opencollective/issues/7542